### ISSUE TITLE
BTT-HDMI7 resolution not correct

### ISSUE THREAD

**Issue by Garag** [bigtreetech/CB2#3]
I have attached a BTT-HDMI7 via HDMI to the board. The board runs from SD Card the  current image (CB2_Debian11_Klipper_kernel4.19_20240422). As described in the user manual i have configured HDMI display.

Linux starts and  i can see boot messages on the screen and later KlipperScreen. But the resolution is not correct.

It seems that Linux uses a resolution of 1280x720 but the display has only 1024x600.

How can i set the correct resolution?

Kernel Message:
rockchip-vop2 fe040000.vop: [drm:vop2_crtc_atomic_enable] Update mode to 1280x720p0, type: 11 for vp0

**Comment by zhangyumingsh** [comment]
> I have attached a BTT-HDMI7 via HDMI to the board. The board runs from SD Card the current image (CB2_Debian11_Klipper_kernel4.19_20240422). As described in the user manual i have configured HDMI display.
> 
> Linux starts and i can see boot messages on the screen and later KlipperScreen. But the resolution is not correct.
> 
> It seems that Linux uses a resolution of 1280x720 but the display has only 1024x600.
> 
> How can i set the correct resolution?
> 
> Kernel Message: rockchip-vop2 fe040000.vop: [drm:vop2_crtc_atomic_enable] Update mode to 1280x720p0, type: 11 for vp0


Try flashing the image again

**Comment by Garag** [comment]
The problem is with the V1.0.4 image solved. The display just works.
Thank you.

**Comment by KricorJunior** [comment]
Hello. Good morning.

I'm having a similar problem with my BTT HDMI7.

I'm using the M4P with the CB2 and the CB2_Debian12_Klipper_kernel6.1_20250821.img image (directly on the CB2, not on the SD card).

My resolution insists on staying at 1280x720.

I've reinstalled klipperscreen several times using KIAUH, and the only protocol that showed anything on the monitor was Wayland (still with the incorrect resolution). Installing klipperscreen via xORG, the screen goes black and the log doesn't detect the monitor.

Searching the internet, I saw some information about forcing the resolution in the armbianEnv.txt file (with extraargs), but even so, the hardware insists on not working.

[KlipperScreen.log](https://github.com/user-attachments/files/24033617/KlipperScreen.log)

run command: dmesg | grep -i error on SSH

<img width="616" height="113" alt="Image" src="https://github.com/user-attachments/assets/1c8ec98a-b7e2-4937-90da-bd3f597bba56" />

<img width="720" height="1280" alt="Image" src="https://github.com/user-attachments/assets/40e47f15-de81-403d-9e67-51e7756e0eca" />

thanks for any help
